apps/angular/src/core/interceptors/api-key.interceptor.ts

```diff
@@ -0,0 +1,21 @@
+import { HttpEvent, HttpHandlerFn, HttpRequest } from '@angular/common/http';
+import { inject } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { AppConfigService } from '../services/app-config.service';
+
+/**
+ * Add 'Api-Key' header to request.
+ * @param req - Request.
+ * @param next - Next interceptor.
+ */
+export function apiKeyInterceptor(req: HttpRequest<unknown>, next: HttpHandlerFn): Observable<HttpEvent<unknown>> {
+	const appConfigService = inject(AppConfigService);
+	const apiKeyReq = req.clone({
```

Comment:
Try to avoid contractions. See our [wiki](https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#avoid-contractions) for additional details:
```suggestion
	const apiKeyRequest = req.clone({
```